pkg/datapath/orchestrator/localnodeconfig.go

```diff
@@ -99,5 +99,6 @@ func newLocalNodeConfig(
 		IPv4PodSubnets:               cidr.NewCIDRSlice(config.IPv4PodSubnets),
 		IPv6PodSubnets:               cidr.NewCIDRSlice(config.IPv6PodSubnets),
 		XDPConfig:                    xdpConfig,
+		RoutingMode:                  config.RoutingMode,
```

Comment:
@ldelossa stumbled over this randomly - why couldn't we use `EnableEncapsulation` instead that gets passed a few lines up?

Reply:
We could have, I missed that field 🤦

Reply:
-> https://github.com/cilium/cilium/pull/36932 let's clean it up :)